addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -3046,8 +3045,8 @@ options.registry.ConditionalVisibility = options.Class.extend({
                     delete targetEl.dataset[attribute.saveAttribute];
                     delete targetEl.dataset[`${attribute.saveAttribute}Rule`];
                 }
-                this.trigger_up('snippet_option_visibility_update');
             }
+            this.trigger_up('snippet_option_visibility_update', {show: true});
```

Comment:
@younn-o I am confused: is this line useful at all?

Reply:
It is there tag the snippet as invisible/visible and update the invisible DOM